test/SlowTests/Corax/CoraxSlowQueryTests.cs

```diff
@@ -172,13 +172,13 @@ public void NgramSuggestionTest(string searchEngine)
                         .SuggestUsing(x => x.ByField(y => y.Name, "Mett").WithOptions(new SuggestionOptions
                         {
                             PageSize = 10,
-                            Accuracy = 0.1f,
+                            Accuracy = 0.8f,
                             Distance = StringDistanceTypes.NGram
                         }))
                         .Execute();
 
                     Assert.Equal(1, suggestionQueryResult["Name"].Suggestions.Count);
-                    Assert.Equal("matt", suggestionQueryResult["Name"].Suggestions[0]);
+                    Assert.Equal("maciej", suggestionQueryResult["Name"].Suggestions[0]);
```

Comment:
If you run this test on Lucene (by decorating it with `[SearchEngineClassData(SearchEngineType.Lucene)]` then you get "maciej" or "matt"?

Reply:
You get `matt` 
https://github.com/ravendb/ravendb/pull/13696#issuecomment-1051005631

Reply:
As I see the tests passed on both engines now.

Reply:
Yes because I modified the test to be a little more forgiving